style/styles_stories.css

```diff
@@ -0,0 +1,267 @@
+@media (max-width: 1085px) {
+  .stories-item {
+    width: 280px;
+  }
+
+  .stories-item__pic {
+    width: 290px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .stories {
+    padding-top: 78px;
+  }
+
+  .stories-item {
+    width: 277px;
+  }
+
+  .stories-item__pic {
+    width: 277px;
+    height: 147.5px;
+  }
+
+  .pic-first {
+    display: inline;
+  }
+
+  .pic-first-big {
+    display: none;
+  }
+
+  .stories-item__article {
+    font-size: 13px;
+    line-height: 16px;
+    padding-bottom: 24px;
+  }
+
+  .stories-item__link::after {
+    content: url("../pic/arrow_stories1024.png");
+    padding-left: 0;
+  }
+}
+
+@media (max-width: 940px) {
+  .stories-item {
+    width: 230px;
+  }
+
+  .stories-item__pic {
+    width: 230px;
+    height: 120px;
+  }
+}
+
+@media (max-width: 768px) {
+  .stories {
+    background-color: #000;
+    color: #fff;
+    padding-top: 26px;
+  }
+
+  .stories__title {
+    font-size: 30px;
+    margin-bottom: 42px;
+  }
+
+  .item-third {
+    display: none;
+  }
+
+  .stories__block {
+    justify-content: center;
+  }
+
+  .item-first {
+    width: 331px;
+    padding-right: 10px;
+  }
+
+  .item-second {
+    width: 312px;
+    padding-left: 28px;
+  }
+
+  .stories-item__pic {
+    width: 312px;
+    height: 166px;
+    padding-bottom: 28px;
+  }
+
+  .stories-item__title {
+    font-size: 25px;
+    padding-bottom: 8px;
+  }
+  
+  .stories-item__article {
+    font-size: 20px;
+    padding-bottom: 36px;
+    width: 313px;
+    line-height: 25px;
+  }
+
+  .stories-item__link {
+    color: #fff;
+    font-size: 20px;
+  }
+
+  .stories-item__link::after {
+    content: url("../pic/arrow_stories768.png");
+    padding-left: 12px;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+  }
+}
+
+@media (max-width: 700px) {
+  .stories-item {
+    width: 280px;
+  }
+
+  .stories-item__pic {
+    width: 280px;
+  }
+
+  .stories-item__title {
+    font-size: 23px;
+  }
+
+  .stories-item__article {
+    font-size: 17px;
+  }
+
+  .stories-item__link {
+    font-size: 16px;
+  }
+}
+
+@media (max-width: 655px) {
+  .item-second {
+    display: none;
+  }
+}
+
+@media (max-width: 375px) {
+  .stories {
+    padding-top: 25px;
+  }
+
+  .stories__title {
+    font-size: 20px;
+    margin-bottom: 26px;
+  }
+
+  .stories__block {
+    padding-left: 11px;
+  }
+
+  .stories-item {
+    width: 279px;
+  }
+
+  .stories-item__pic {
+    width: 279px;
+    height: 148px;
+    padding-bottom: 27px;
+  }
+
+  .stories-item__title {
+    font-size: 20px;
+    padding-bottom: 10px;
+  }
+
+  .stories-item__article {
+    width: 278px;
+    font-size: 14px;
+    line-height: 17.7px;
+    padding-bottom: 26px;
+  }
+
+  .stories-item__link {
+    font-size: 12px;
+  }
+
+  .stories-item__link::after {
+    content: url("../pic/arrow_stories375.png");
+    padding-left: 11px;
+  }
+}
+
+@media (max-width: 320px) {
+  .stories__block {
+    padding-left: 12px;
+  }
+
+  .stories__title {
+    padding-left: 0;
+  }
+} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)